app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { getAssetPath } from '@18f/identity-assets';
+import { FullScreen } from '@18f/identity-components';
+import AcuantContext from '../context/acuant';
+
+function FullScreenLoadingSpinner({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+  return (
+    <FullScreen ref={fullScreenRef} label={fullScreenLabel} onRequestClose={onRequestClose}>
+      <img
+        src={getAssetPath('loading-badge.gif')}
+        alt=""
+        width="144"
+        height="144"
+        className="acuant-capture-canvas__spinner"
+      />
+    </FullScreen>
+  );
+}
+
+function AcuantSelfieCaptureCanvas({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+  const { isReady } = useContext(AcuantContext);
+  return !isReady ? (
+    <FullScreenLoadingSpinner
+      fullScreenRef={fullScreenRef}
+      onRequestClose={onRequestClose}
+      fullScreenLabel={fullScreenLabel}
+    />
+  ) : (
+    <div id="acuant-face-capture-container" />
```

Comment:
Do we need the ID? And if so, should we consider using `useInstanceId` to guarantee a unique ID, which is generally a good practice for React components to avoid assuming there's only ever a single instance of the component.

https://github.com/18F/identity-idp/blob/main/app/javascript/packages/react-hooks/use-instance-id.ts

Reply:
I extracted this id, named it, and added a comment that should clarify ([commit](https://github.com/18F/identity-idp/pull/9580/commits/f14748eaa6bfa35ff1d9d404a7416c53af8360fa)).